_rules/link-in-context-descriptive-5effbb.md

```diff
@@ -337,6 +347,7 @@ This `a` element is not a [semantic][semantic role] `link`, because it has no `h
 ```
 
 [accessible name]: #accessible-name 'Definition of Accessible Name'
+[accessible description]: https://www.w3.org/TR/wai-aria-1.2/#dfn-accessible-description 'Definition of accessible description'
```

Comment:
Should we have our own def of it (like for accessible name?), I'm notably thinking we should link it to the "accessible name and description computation" document.

Reply:
Added a definition of accessible description